cmd/osbuild-composer-dbjobqueue-tests/main_test.go

```diff
@@ -64,6 +64,7 @@ func TestJobQueueInterface(t *testing.T) {
 
 	t.Run("maintenance-query-jobs-before", wrap(testJobsUptoByType))
 	t.Run("maintenance-delete-job-results", wrap(testDeleteJobResult))
+	t.Run("close", wrap(testClose))
```

Comment:
This needs to be the last test in the suite, I am lazy yeah. Let me know how to implement this in a nicer way, perhaps to reopen the connection afterwards? :)

Reply:
Two ideas:

- put a comment before this subtest to indicate that it must be run last (lazier, gets the job done)
- make a completely separate test-case that spins up its own queue and then immediately close it (cleaner)

I'm fine with both. :)

Reply:
I was thinking exactly the same. Going for the cleanest solution.

I cannot figure out how to actually run those tests (I am still new to Go), so force-pushed blindly with fingers-crossed. If it fails, teach me please how to run them (this appear to be some kind of test suite, how do I trigger it)?